packet.go

```diff
@@ -109,6 +112,11 @@ func parsePacket(i *astikit.BytesIterator) (p *Packet, err error) {
 		}
 	}
 
+	// Use optional packet filter
```

Comment:
Could you replace this comment with `// Skip packet`?

Reply:
Done